components/test_pd/src/server.rs

```diff
@@ -136,10 +135,7 @@ fn hijack_unary<F, R, C: PdMocker>(
             );
         }
         _ => {
-            let status = RpcStatus::new(
-                RpcStatusCode::UNIMPLEMENTED,
-                Some("Unimplemented".to_owned()),
-            );
+            let status = RpcStatus::new(RpcStatusCode::UNAVAILABLE, Some("Unavailable".to_owned()));
```

Comment:
why change this?

Reply:
for testing purpose

Reply:
I don't think it's a good idea. How about constructing the error in the failpoint directly?